.gitignore

```diff
@@ -35,3 +35,5 @@ dist/*
 .envrc
 .flox/
 .vscode/private.env
+
+setvars.sh
```

Comment:
let's move this into your personal gitignore; we don't use this file anywhere in the project.

Reply:
ok, will do